src/playercards/Tarotcard.ttslua

```diff
@@ -0,0 +1,19 @@
+-- context menu to manually fix rotation
+function onLoad()
+  self.addContextMenuItem("rotate preview", rotatePreview)
+  self.addContextMenuItem("rotate card+preview", function()
```

Comment:
If one is going to be a separate function, this one should be as well.  Reads very weird otherwise

Reply:
Updated